src/components/model-form/model-form.styles.js

```diff
@@ -1,68 +1,39 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { formStyles } from '../../configs/styles';
 
-export const useStyles = makeStyles((theme) => ({
-  modelItemUpdate: {
-    display: 'flex',
-    flexDirection: 'column',
-    margin: '20px 0'
-  },
-  modelDetails: {
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'space-around'
-  },
-  saveButton: {
-    margin: theme.spacing(2)
-  },
-  returnButton: {
-    margin: theme.spacing(2),
-    marginRight: 0
-  },
-  textField: {
-    margin: '10px 5px'
-  },
-  tabs: {
-    backgroundColor: 'white',
-    '& span.MuiTab-wrapper': {
-      color: '#3F51B5'
+export const useStyles = makeStyles((theme) => {
+  const {
+    DETAILS_STYLES,
+    SAVE_BUTTON_STYLES,
+    RETURN_BUTTON_STYLES,
+    TEXT_FIELD_STYLES,
+    CONTROLS_BLOCK_STYLES,
+    TABS_STYLES,
+    LARGE_STYLES,
+    ATTACH_FILE_STYLES,
+    IMAGE_UPLOAD_STYLES,
+    ITEM_UPDATE_STYLE,
+    INPUT_ERROR_STYLES,
+    IMAGE_UPLOAD_CONTAINER_STYLES,
+    IMAGE_NAME_STYLE
```

Comment:
Nice to see that everything is put together now! The only thing I recognize is that maybe we should have "STYLES" like everywhere else.

```suggestion
    IMAGE_NAME_STYLES
```

Reply:
Fixed